### ISSUE TITLE
reproducibility of FFHQ-1K Figure 7 of paper 

### ISSUE THREAD

**Issue by nupurkmr9** [NVlabs/stylegan2-ada-pytorch#78]
Hi,
When I measured FID of provided _ffhq1k-paper256-ada.pkl_ model using a random 1K subset of FFHQ png images resized to 256x256 using dataset_tool.py, I got the FID 40.34 instead of the reported score 21.29 in paper. I used the provided script "calc_metrics.py" for computing FID with "fid50k_full" metric and no truncation. Is this expected if FID is computed using a random subset of FFHQ as real distribution? 

Also,  _paper256_ config trains the model using 8 GPUs with 64 mini batch. Can you provide any recommendations on what hyper-parameters need to be changed if I want to train a similar FID model on 2 GPUs? Will the "batch_gpu" remain same which will lead to 16 mini-batch size or the maximum possible 32 mini batch size. Will using cfg=='auto' result in the best model? 

Thanks!!
